forge-gui/res/cardsfolder/upcoming/syr_joshua_and_syr_saxon.txt

```diff
@@ -0,0 +1,8 @@
+Name:Syr Joshua and Syr Saxon
+ManaCost:G W
+Types:Legendary Creature Human Knight
+PT:2/3
+K:Double team
+S:Mode$ IgnoreLegendRule | ValidCard$ Permanent.namedSyr Joshua and Syr Saxon | IsPresent$ Permanent.namedSyr Joshua and Syr Saxon | PresentCompare$ EQ2 | Description$ If you control exactly two legendary permanents named Syr Joshua and Syr Saxon, the "legend rule" doesn't apply to them.
```

Comment:
Hmm, I think the IsPresent should contain `Legendary`

There is potential where you might have two "Syr J and Syr S" but one isn't legendary... Double Major for example

Reply:
Oh good catch. It even says "legendary permanents named" in the oracle